spanner/google/cloud/spanner/client.py

```diff
@@ -154,6 +154,7 @@ def instance_admin_api(self):
         """Helper for session-related API calls."""
         if self._instance_admin_api is None:
             self._instance_admin_api = InstanceAdminClient(
+                credentials=self.credentials,
```

Comment:
@lukesneeringer How long has support for `credentials` been in the GAPIC surfaces?

Should we change how [it's done][1] in Pub / Sub?

[1]: https://github.com/GoogleCloudPlatform/google-cloud-python/blob/b65733717492844c4ff7dd08f18636ec2ce30de4/pubsub/google/cloud/pubsub/_gax.py#L669-L676